nrf_802154_sl/Kconfig

```diff
@@ -17,6 +17,8 @@ config NRF_802154_SL
 	select MPSL
 	select ZERO_LATENCY_IRQS
 	select DYNAMIC_INTERRUPTS
+	select SENSOR
```

Comment:
Why is it selected only for `NRF_802154_SL`, not for `NRF_802154_RADIO_DRIVER`? I thought RSSI correction is implemented in `NRF_802154_RADIO_DRIVER` regardless SL mode is selected.

Reply:
Do you mean only `select SENSOR` or the two added lines?

Reply:
I think both are necessary to do RSSI correction, right? Because of that I'm thinking if both belong here or in `NRF_802154_RADIO_DRIVER`

Reply:
Right. However, currently there are two sources of temperature: `TEMP_NRF5` and `TEMP_NRF5_MPSL`. Those two cannot be used simultaneously. `NRF_802154_SL` is used with `MPSL`, hence this option is selected here. For `NRF_802154_SL_OPENSOURCE`, `TEMP_NRF5_MPSL` cannot be used and `TEMP_NRF5` selection is to be introduced to Zephyr by a separate PR.

Reply:
Do I understand correctly `...SL_OPENSOURCE` selects `SENSOR` and `TEMP_NRF5_MPSL`, while `...SL` selects `SENSOR` and `TEMP_NRF5`?

Reply:
It is exactly like that.